server/graphql/v2/query/SearchQuery.ts

```diff
@@ -53,47 +45,15 @@ const SearchQuery = {
     const account = args.account && (await fetchAccountWithReference(args.account));
     const limit = args.defaultLimit;
 
-    // Fall back to Postgres in non-production environments if ElasticSearch is not configured
-    const hasElasticSearch = Boolean(getElasticSearchClient());
-    let useElasticSearch = hasElasticSearch && args.useElasticSearch;
-    if (args.useElasticSearch && !hasElasticSearch && config.env !== 'production') {
-      logger.warn('ElasticSearch is not configured, falling back to Postgres');
-      useElasticSearch = false;
+    if (!getElasticSearchClient()) {
+      throw new Error(
+        config.env === 'development' ? 'Elastic search is not running' : 'The search is temporarily unavailable',
+      );
     }
 
     // Elastic search
-    if (useElasticSearch) {
-      const requestId = getElasticSearchQueryId(remoteUser, host, account, searchTerm);
-      const query = { requestId, limit, searchTerm, account, host };
-      return {
-        results: {
-          accounts: getElasticSearchIndexResolver(req, 'collectives', query),
-          comments: getElasticSearchIndexResolver(req, 'comments', query),
-          expenses: getElasticSearchIndexResolver(req, 'expenses', query),
-          hostApplications: getElasticSearchIndexResolver(req, 'host-applications', query),
-          orders: getElasticSearchIndexResolver(req, 'orders', query),
-          tiers: getElasticSearchIndexResolver(req, 'tiers', query),
-          transactions: getElasticSearchIndexResolver(req, 'transactions', query),
-          updates: getElasticSearchIndexResolver(req, 'updates', query),
-        },
-      };
-    }
-    // This falls back to Postgres as a search engine, which is not efficient and should be used only for dev/debugging.
-    else {
-      const query = { searchTerm, limit, remoteUser, host, account, timeout: args.timeout };
-      return {
-        results: {
-          accounts: getSQLSearchResolver(models.Collective, query),
-          comments: getSQLSearchResolver(models.Comment, query),
-          expenses: getSQLSearchResolver(models.Expense, query),
-          hostApplications: getSQLSearchResolver(models.HostApplication, query),
-          orders: getSQLSearchResolver(models.Order, query),
-          tiers: getSQLSearchResolver(models.Tier, query),
-          transactions: getSQLSearchResolver(models.Transaction, query),
-          updates: getSQLSearchResolver(models.Update, query),
-        },
-      };
```

Comment:
The SQL search was initially implemented for the hackathon and could have been useful in dev, but maintaining feature parity seems too expensive for the narrow usage it will have.